minimal-plugin/src/main/java/io/micronaut/gradle/ApplicationClasspathInspector.java

```diff
@@ -18,12 +18,7 @@
 import org.gradle.api.DefaultTask;
 import org.gradle.api.file.ConfigurableFileCollection;
 import org.gradle.api.file.RegularFileProperty;
-import org.gradle.api.tasks.CacheableTask;
-import org.gradle.api.tasks.InputFiles;
-import org.gradle.api.tasks.OutputFile;
-import org.gradle.api.tasks.PathSensitive;
-import org.gradle.api.tasks.PathSensitivity;
-import org.gradle.api.tasks.TaskAction;
+import org.gradle.api.tasks.*;
```

Comment:
Is this in accordance with the Micronaut project style?

Reply:
No, this should be restored as individual imports